src/components/Dialogs/Dialogs.jsx

```diff
@@ -1,52 +1,46 @@
 import React from 'react';
-import styles from './Dialogs.module.css';
-import DialogItem from "./DialogItem/DialogItem";
-import Message from "./Message/Message";
-import { maxLengthCreator } from '../../utils/validation/validation';
-import { AddMessageForm } from './Message/AddMessageForm';
 import { reduxForm } from 'redux-form';
+
+import DialogItem from './DialogItem/DialogItem';
+import Message from './Message/Message';
+import { AddMessageForm } from './Message/AddMessageForm';
 import Sidebar from '../Sidebar/Sidebar';
 import Navbar from '../Navbar/Navbar'

```

Comment:
did you notice you do too much repetition? here, mostly everywhere
import DialogItem from './DialogItem/DialogItem`
import Message from './Message/Message`
etc

instead, if you'd use a index.jsx file in all of your components, you'll end up:
import DialogItem from './DialogItem` -> automatically calls DialogItem/index.jsx
import Message from './Message` -> automatic Message/index.jsx

please read some teoretical material about this /index import feature,
and also about various file structures, pros and cons of each other